docs: Update candidate information (#67)
guardian/coding-exercises: CANDIDATE_INFORMATION.md

A link will be included on your interview invitation, click it to join.
Once joined, your interviewer will ask you to share your screen.

> **Note**

> > - Enabling your camera is optional.
> > - We do not assess your IDE setup.
> > - We currently forbid the use of GitHub Copilot and similar tools. They make it difficult to perform a meaningful assessment.

Reviewer: Suggest removing the reasoning for forbidding Github Copilot. Most candidates would understand it, or ask about it. We could provide this statement as an guideline explanation in the [PROCESS_REMOTE.md](https://github.com/guardian/coding-exercises/blob/main/PROCESS_REMOTE.md)

```suggestion
> - Turning on your camera is optional.
> - We do not assess your IDE setup.
> - We currently do not allow the use of GitHub Copilot and similar tools.
```

We do not assess your IDE setup.

> Please note that we do not accept use of GitHub co-pilot and similar tools, as it makes it difficult for a meaningful assessment to be performed.

Reviewer 2: I would merge the three last points into a:

```md
> **Note**
> - Enabling your camera is option
> - We do not assess your IDE setup
> - We currently forbid the use of GitHub Copilot and similar code-generation tools. They make it difficult to perform a meaningful assessment.
```
